api-spec/wallet-api.yaml

```diff
@@ -324,6 +369,50 @@ components:
       description: Wallet identifier
       type: string
       format: uuid
+    OrderId:
+      description: Order session payment gateway identifier
+      type: string
+      format: uuid
+    WalletVerifyRequestsResponse:
+      type: object
+      description: Data to perform a wallet verify with payment gateway
+      properties:
+        orderId:
+          $ref: '#/components/schemas/OrderId'
+        details:
+          description: Redirection URL or iframe url according payment method type
+          type: object
+          oneOf:
+            - $ref: '#/components/schemas/WalletVerifyRequestCardDetails'
+            - $ref: '#/components/schemas/WalletVerifyRequestAPMDetails'
+          discriminator:
+            propertyName: type
+            mapping:
+              CARDS: '#/components/schemas/WalletVerifyRequestCardDetails'
```

Comment:
@infantesimone I Removed the enum since it causes wrong generation by openapi-codegen.